### ISSUE TITLE
LoadData error with OMERO 5.3

### ISSUE THREAD

**Issue by lguerard** [CellProfiler/java#33]
Hi,
Since our upgrade to OMERO 5.3 we can't connect to it via the LoadData module in CellProfiler. After discussing with people from OME and reading some issues about it, it seems that it's because prokaryote is only working for 5.2.

Do you guys have any working version for 5.3 available ? Or any version working for all the future versions as well ? I read somewhere about using python readers ?

Thanks for your help.

**Comment by chris-allan** [comment]
I have tested, on Linux, that the current `master` (d44e820ec64e92b836f6b24db9560c2c05fc46d3 ) of this dependency works fine with CellProfiler 3.x and OMERO 5.3.x.

To use it, install CellProfiler from source following one or more of the guides:

 * https://github.com/CellProfiler/CellProfiler/wiki/Source-installation-%28Linux%29

You will then need to Git clone this prokaryote repository somewhere, check out `master`, build the JAR dependency and apply the dependency to your `PYTHONPATH`. For example:

```
pip uninstall prokaryote
git checkout master
mvn clean
mvn
pip install -U -e .
```

Your prokaryote dependency should then look something like this:

```
$ pip freeze | grep prokaryote
-e git+git@github.com:CellProfiler/prokaryote.git@d44e820ec64e92b836f6b24db9560c2c05fc46d3#egg=prokaryote
```

The current `master` branch __WILL NOT__ work with CellProfiler 2.2.x. In fact, if you install the `2.2.0` or `v2.2.1` CellProfiler tag from source you will get a version of prokaryote (2.3.1) that is quite broken. It will __NOT__ start up with this version of prokaryote.

In order to re-instate compatibility and utilize OMERO 5.3.x, until @0x00b1 or @mcquin merge #38, you will need to check out that PR's branch (`ij1-patcher-fix2`), build and deploy similarly to the above:

```
pip uninstall prokaryote
git remote add chris-allan git@github.com:chris-allan/prokaryote.git
git checkout chris-allan/ij1-patcher-fix2
mvn clean
mvn
pip install -U -e .
```

Your prokaryote dependency should then look something like this:

```
$ pip freeze | grep prokaryote
-e git+git@github.com:CellProfiler/prokaryote.git@853963b8752d2ede8d17b30f425347b689ac5ce6#egg=prokaryote
```

This `ij1-patcher-fix2` branch will work both CellProfiler 2.2.x and CellProfiler 3.x. If you install from source you will get a version of prokaryote (2.3.1) that is quite broken.

/cc @ehrenfeu

**Comment by lguerard** [comment]
Hi @chris-allan 

Thanks a lot for your work. I can confirm that the `ij1-patcher-fix2` works as expected in both Linux and Windows with CellProfiler 2.2.0.

**Comment by lguerard** [comment]
Hi @chris-allan,

I just tested it on both Windows and Linux and everything works again with CP 2.2.0. Thanks so much for your help ! We'll try even more and let you know.

Thank you again :)

**Comment by chris-allan** [comment]
Glad to hear it. This issue can probably be closed then as @0x00b1 has already made a 2.3.2 release and closed #39.

**Comment by 0x00b1** [comment]
Awesome!
